update-documentation-use.md (#20485)
Azure/azure-sdk-for-js: documentation/how-to-use.md

1. Create a ts file (free name and copy follow code into this file) eg: test_1.ts
    eg：
    ```

>         const DefaultAzureCredential = require("@azure/identity");

Reviewer: ```suggestion
        const { DefaultAzureCredential } = require("@azure/identity");
```

        const { TargetManagementClient } = require("@azure/arm-target");  // must same as dependencies
        const subscriptionId = process.env["AZURE_SUBSCRIPTION_ID"];

        async function main(){
          const credentials = new DefaultAzureCredential();
          const client = new TargetManagementClient(credentials,subscriptionId);   // must same as dependencies
          for await (const item of client.operations.list()){    // you can test that you need test operation
            console.log(item)  
          }
        }
        main().catch(err => {
          console.log(err);
        });
    ```   
    A few points, you need to pay attention here:  

>     In the example, we are using DefaultAzureCredential, you can also try other way such as InteractiveBrowserCredential

Reviewer: Keep the Service Principal way

>     In the example, we are using DefaultAzureCredential, you can also try other way such as InteractiveBrowserCredential

Reviewer: ```suggestion
    In the example, we are using DefaultAzureCredential, you can also try other way such as Service Principal
```

    ```

>         const credentials = new DefaultAzureCredential();;

Reviewer: ```suggestion
        const credentials = new ClientSecretCredential(tenantId, clientId, clientSecret);
```
